src/test/resources/unit-test-config.yml

```diff
@@ -0,0 +1,119 @@
+#
+# Http-interfaces
+#
+server:
+  applicationContextPath: /
+  adminContextPath: /
+  applicationConnectors:
+    - type: http
+      port: 20300
+  adminConnectors:
+    - type: http
+      port: 20301
+
+ingestFlow:
+  import:
+    inbox: data/import/inbox
```

Comment:
Unit test must not use `data/` but instead `test/<test-class-name>` as working area. See: https://dans-knaw.github.io/dans-module-archetype/common-practices/#testing